package.json

```diff
@@ -81,46 +102,41 @@
     "babel-cli": "6.26.0",
     "babel-core": "6.26.0",
     "babel-eslint": "8.0.2",
+    "babel-jest": "21.2.0",
     "babel-loader": "7.1.2",
+    "babel-plugin-dynamic-import-node": "1.2.0",
     "babel-plugin-import": "1.6.2",
-    "babel-plugin-istanbul": "4.1.5",
+    "babel-plugin-transform-es2015-modules-commonjs": "6.26.0",
     "babel-plugin-transform-import-extension-jsx-to-js": "0.1.0",
     "babel-polyfill": "6.26.0",
     "babel-preset-es2015": "6.24.1",
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-0": "6.24.1",
-    "coveralls": "3.0.0",
+    "canvas-prebuilt": "1.6.5-prerelease.1",
     "css-loader": "0.28.7",
     "enzyme": "3.1.1",
     "enzyme-adapter-react-16": "1.0.4",
     "eslint": "4.11.0",
     "eslint-plugin-html": "4.0.0",
     "eslint-plugin-markdown": "1.0.0-beta.7",
     "eslint-plugin-react": "7.4.0",
-    "expect.js": "0.3.1",
```

Comment:
Yay!

Reply:
I thought the same, even though I like the public API of expoect a bitt more thant the jest one. But the pros are way more important!